std/http/server_test.ts

```diff
@@ -655,7 +655,15 @@ Deno.test({
       port: 8124,
       transport: "tcp",
     };
-    assertEquals(expected, server.listener.addr);
+    assertEquals(server.listener.addr, expected);
     server.close();
   },
 });
+
+Deno.test({
+  name: "server._parseAddrFromStr() port 80",
+  fn: (): void => {
+    const addr = _parseAddrFromStr(":80");
+    assertEquals(addr.port, 80);
```

Comment:
This test case doesn't seem to cover the changes that were made... the changes seem to suggest it should default to port 80 if no port is provided.

Reply:
Well, the issue is that whether the default port for http (80) or no port at all is supplied to `_parseAddrFromStr`, after the host string is passed through the `URL` constructor, the port is set to the empty string. i.e.:
```
_parseAddrFromStr(':80');
_parseAddrFromStr(':');
```
both return
```
{ hostname: "", port: 0 }
```
Should I make the test cover both cases?

Reply:
I see, it's okay as is.